example/dapp-store/config.yaml

```diff
@@ -1,25 +1,25 @@
 publisher:
   name: Publisher (contains apps)
-  address: ''
+  address: GcRW5MzJ7GHpoxWp73UEVTX3F6vq54FpnmqwG3wvdKdp
```

Comment:
Did you mean to commit actual addresses to this file? If so, are they devnet or mainnet addresses?